recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
Dot has been recognised as a valid character in conda package names:
http://conda.pydata.org/docs/building/pkg-name-conv.html

If there are no other blocking issues, can someone accept this recipe? Thanks.